NWHarvest.Web/Controllers/AccountController.cs

```diff
@@ -563,7 +563,68 @@ private void RegisterViewBag()
                     Selected = true
                 }
             };
+
+            ViewBag.Counties = GetCounties();
+        }
+
+        private List<SelectListItem> GetCounties()
+        {
+            List<string> counties =
+                new List<string>
+                {
+                    "Unknown",
```

Comment:
Is this necessary?

Reply:
Rachel thought it will be good to have this option in case Growers don't know what county they are in.

Reply:
I see.  If it's necessary for user to have this option rather than have them look it up, I suggest placing it as the last item on the list.  This will force those users who are unsure to go through the list before selecting Unknown.  Since the county field will eventually be used as one way to filter listings, user with unknown county will be an issue.

Reply:
Makes sense. Update the list to have "Unknown" as the last option.